<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileParserDialog</class>
 <widget class="QDialog" name="FileParserDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Разбор файла с кадрами</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="file_label">
     <property name="text">
      <string>Файл с кадрами</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="file_lineEdit">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>C:/Project/data/temp/data.dat</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="file_pushButton">
     <property name="text">
      <string>Выбрать</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="start_pushButton">
     <property name="text">
      <string>Старт</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="path_label">
     <property name="text">
      <string>Папка для кадров</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="path_lineEdit">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>C:/Project/data/temp</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="path_pushButton">
     <property name="text">
      <string>Выбрать</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="stop_pushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Стоп</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
